tf2onnx/onnx_opset/tensor.py

```diff
@@ -1248,3 +1248,142 @@ def version_10(cls, ctx, node, **kwargs):
         target_dtype = TensorProto.INT64
         if seq_len_dtype != target_dtype:
             ctx.insert_new_node_on_input(node, "Cast", node.input[1], to=target_dtype)
+
+
+@tf_op("ReverseV2")
+class ReverseV2:
+    @classmethod
+    def version_9(cls, ctx, node, **kwargs):
+        raise NotImplementedError("ReverseV2 is not supported to convert in OPSET 9,"
+                                  " if possible please try using OPSET>=10 instead.")
+
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+
+        axes_node = node.inputs[1]
+        axes = axes_node.get_tensor_value() # Is a Python list.
+        len_axes = len(axes)
+
+        # Store input and output parameters of the ReverseV2 node.
+        rv2_in_names = [node.input[0]]
+        rv2_out_name = node.output
+        output_shape = ctx.get_shape(node.output[0])
+        len_shape = len(output_shape)
+        rv2_node_name = node.name
+
+        # Remove ReverseV2 node from graph.
+        ctx.remove_node(rv2_node_name)
+
+        # List to keep track of input names.
+        inputs = [rv2_in_names]
```

Comment:
really need a list to track all inputs? It seems the last inputs is enough for us to connect the subgraph.